docs/domainmetadata.rst

```diff
@@ -18,6 +18,15 @@ documentation.
 Apart from raw SQL statements, setting domain metadata can be done with
 ``pdnsutil set-meta`` and retrieving metadata is done with ``pdnsutil get-meta``.
 
+The following options can not be edited via the metadata endpoint:
```

Comment:
```suggestion
The following options can not be edited via the HTTP API metadata endpoint:
```

Reply:
I did not use the 'Commit suggestion' option, but added the text, when I was editing the file.